### PR TITLE
Update keycloak, superset envs

### LINKED ISSUE #22: [Capacity Building Issue] Missing configurations that make instance display a blank white page.
## Describe the issue

The preview instance deployed [here](https://preview-mhealth.ddc-malaria.org/), does not load up properly.
Steps:

1. Navigate to https://preview-mhealth.ddc-malaria.org/
2. Wait for the web application to load up.

You only see a white blank page instead of the login view.

## How is this expected to work?
The web application should load, and redirect the user to the home page if logged in, or to the keycloak login page.

Objectives
 - [ ] To find and fix either the missing or mis-configured envs.

## Screenshots
image

## Please share other relevant information about the issue
The screenshot above shows that it could be something to do with the content security policies, these are currently configured in the express server. see [this](https://github.com/onaio/express-server/pull/29/files)

### LINKED ISSUE #28: Disable Certbort in reveal-web deployment configs
The certbot role is currently enabled for reveal-web thailand playbooks.

We should update reveal web ansible configuration for preview and production deployment to use the cert provided by the client. This entails disabling running the certbot roles.

### PR DESCRIPTION
**Pull request by peterMuriuki**
- Update inventory variables for reveal-web.
- Bump web and express version tags

- resolves #28 
- closes #22

### CODE REVIEW

File: ansible/inventories/production/opensrp-stack/group_vars/all/vars.yml [hunk 1]
```diff
@@ -75,3 +75,7 @@ reveal_new_db: "192.168.100.252"
 opensrp_rabbitmq_host: "localhost"
 opensrp_rabbitmq_username: "opensrp_reveal"
 opensrp_rabbitmq_password: "{{ vault_opensrp_rabbitmq_password }}"
+
+reveal_ssl_dir: "../Reveal-Thailand/ansible/inventories/preview/opensrp-stack/files/ssl"
```
Comment (bennsimon) [inline ansible/inventories/production/opensrp-stack/group_vars/all/vars.yml:79]: Shouldn't this point to production certs?

File: ansible/inventories/production/opensrp-stack/group_vars/all/vars.yml [hunk 2]
```diff
@@ -75,3 +75,7 @@ reveal_new_db: "192.168.100.252"
...
+reveal_ssl_cert_file: "star.ddc-malaria.org.crt"
+reveal_ssl_key_file: "star.ddc-malaria.org.key"
```
Comment (bennsimon) [inline ansible/inventories/production/opensrp-stack/group_vars/all/vars.yml:81]: Update the certs to match production ssl files.

**Review by bennsimon — CHANGES_REQUESTED** [review]

File: ansible/inventories/production/opensrp-stack/group_vars/all/vars.yml (hunk 1 above)
Comment (p-netm) [reply ansible/inventories/production/opensrp-stack/group_vars/all/vars.yml:79]: yeah good catch, some lazy copy paste

**Review by bennsimon — CHANGES_REQUESTED** [review]
Kindly update the prod SSL certificates.

**Review by bennsimon — APPROVED** [review]

**Pull request merged**